src/store.c

```diff
@@ -767,8 +787,8 @@ int cs_replace(struct clip_store *cs, enum cs_iter_direction direction,
     size_t nr_lines = first_line(content, line);
     uint64_t hash = djb64_hash(content);
     cs_snip_update(snip, hash, line, nr_lines);
-    ret = cs_content_add(cs, hash, content);
-    if (ret) {
+    ret = cs_content_add(cs, hash, content, 0);
```

Comment:
This is a good example of why `bool` variables are generally not a good idea: this contextless `0` now requires going to `cs_content_add` to find out what it's for.

Instead:

```
enum cs_dupe_policy { CS_DUPE_KEEP_ALL, CS_DUPE_KEEP_LAST };
```

The argument name can be `dupe_policy`.